.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "editor.formatOnSave": true
```

Comment:
this is something I'd rather configure in a pre-commit rule or in your specific editor, it seems a bit of a weird place to enforce this since we don't enforce that all developers use vsCode.